### ISSUE TITLE
Item 21 Python 2 example

### ISSUE THREAD

**Issue by bslatkin** [bslatkin/effectivepython#29]
The error message in the book from calling the keyword-only function in Python 2 with too many positional arguments is wrong. It should be:

```
Trying to pass keyword-only arguments by position won’t work, just like in Python 3.
safe_division_d(1, 0, False, True)
>>>
TypeError: safe_division_d() takes 2 positional arguments but 4 �were given
Trying to pass unexpected keyword arguments also won’t work.
safe_division_d(0, 0, unexpected=True)
>>>
TypeError: Unexpected **kwargs: {'unexpected': True}
```

**Comment by bslatkin** [comment]
Fixed in example code in f892cd116d27605c57c625be3b79e317759365cb

**Comment by bslatkin** [comment]
This will be fixed in the next printing.
